ai/vllm-deployment/README.md

```diff
@@ -116,6 +117,35 @@ Expected output (or similar):
 
 - Update `MODEL_ID` within deployment manifest to serve different model (ensure Hugging Face access token contains these permissions).
 - Change the number of `vLLM` pod replicas in the deployment manifest.
+
+---
+
+## Cloud Provider Differences
+
+Node selectors make sure vLLM pods land on Nodes with the correct GPU, and they are the main difference among the cloud providers. The following are node selector examples for three cloud providers.
+
+- GKE
+  This `nodeSelector` uses labels that are specific to Google Kubernetes Engine.
+  - `cloud.google.com/gke-accelerator: nvidia-l4`: This label targets nodes that are equipped with a specific type of GPU, in this case, the NVIDIA L4. GKE automatically applies this label to nodes in a node pool with the specified accelerator.
+  - `cloud.google.com/gke-gpu-driver-version: latest`: This label ensures that the pod is scheduled on a node that has the latest stable and compatible NVIDIA driver, which is automatically installed and managed by GKE.
+  ```yaml
+  nodeSelector:
+    cloud.google.com/gke-accelerator: nvidia-l4
+    cloud.google.com/gke-gpu-driver-version: latest
+  ```
+- EKS
```

Comment:
This is a valuable addition for making this example useful on other clouds. My one question is about long-term maintenance. Since our team's expertise is primarily with GKE, how can we ensure the configurations for other platforms stay up-to-date? Perhaps we could add a note welcoming community contributions to maintain them?

Reply:
This is a good question that I don't have an answer to at the moment. In a separate PR, I could add a CONTRIBUTING.md, which lays out our expectations for maintenance from each cloud provider (and/or bare metal). Let's discuss this in person.